crypto/crypto.h

```diff
@@ -0,0 +1,121 @@
+#ifndef SECVARCTL_CRYPTO_H
+#define SECVARCTL_CRYPTO_H
+
+#ifdef OPENSSL
+
+#include <openssl/obj_mac.h>
+#define CRYPTO_MD_SHA1 NID_sha1
+#define CRYPTO_MD_SHA224 NID_sha224
+#define CRYPTO_MD_SHA256 NID_sha256 
+#define CRYPTO_MD_SHA384 NID_sha384
+#define CRYPTO_MD_SHA512 NID_sha512
+
+#else
```

Comment:
Personal preference: maybe set a symbol for `MBEDTLS` in the Makefile/cmake, even if it is the default. I don't suspect we'll need to port to yet another crypto library, but I am a fan of avoiding vague "else"s.